### PR TITLE
[SW-393] Allow to configure timeout for clouding up in external mode …

### PR DESCRIPTION
**Pull request by jakubhava**
…from sparkling water

### CODE REVIEW

File: core/src/main/scala/org/apache/spark/h2o/backends/external/ExternalBackendConf.scala [hunk 1]
```diff
@@ -129,6 +135,9 @@ trait ExternalBackendConf extends SharedBackendConf {
 
 object ExternalBackendConf {
 
+  /** Timeout for starting h2o external cluster */
+  val PROP_EXTERNAL_CLUSTER_START_TIMEOUT = ("spark.ext.h2o.cluster.start.timeout", 120)
```
Comment (mmalohlava) [inline core/src/main/scala/org/apache/spark/h2o/backends/external/ExternalBackendConf.scala]: What is unit ?

Comment (jakubhava) [inline core/src/main/scala/org/apache/spark/h2o/backends/external/ExternalBackendConf.scala]: Seconds, good catch! I will comment it properly

**Comment by mmalohlava** [comment]
@jakubhava Ups, there is a conflict. Pls could you pls update this PR.

**Comment by jakubhava** [comment]
@mmalohlava should be good now

**Pull request merged**